lib/show.rb

```diff
@@ -0,0 +1,16 @@
+class Show
+
+  attr_reader :name, :creator, :characters
+
+  def initialize(name, creator, characters)
+    @name = name
+    @creator = creator
+    @characters = characters
+  end
+
+  def total_salary
+    @characters.map(&:salary)
+    .map(&:to_i) # in case salary is in string
+    .sum
+  end
```

Comment:
Good job in here; be sure to indent lines of code that continue from a previous line, so something like this:
```ruby
  def total_salary
    @characters.map(&:salary)
      .map(&:to_i) # in case salary is in string
      .sum
  end
```